lib/cli.js

```diff
@@ -21,82 +21,70 @@ var fs = require('fs');
 var getGitFolder = require('./getGitFolder');
 var validateMessage = require('../index');
 
-// hacky start if not run by mocha :-D
-// istanbul ignore next
-if (process.argv.join('').indexOf('mocha') === -1) {
-  var bufferToString = function (buffer) {
-    var hasToString = buffer && typeof buffer.toString === 'function';
-    return hasToString && buffer.toString();
-  };
+var commitMsgFileOrText = process.argv[2];
+var commitErrorLogPath = process.argv[3];
 
-  var getFileContent = function (filePath) {
```

Comment:
The link is not here for Angular but for the arguments. Ok for your point on the style debate.
The fact is, actually most of the project is written with functions declarations. I like the advantages that comes with.
With this PR I choose consistency.
So you ask me to change all the files to functions expressions and I ask you what's the advantages ? :-)